src/styling/Constants.js

```diff
@@ -20,7 +20,8 @@ const constants = {
         accentPrimary: '#0093D0',
         lightAccentPrimary: '#AFD7F2',
         accentSecondary: '#F1592A',
-        accentTertiary: '#E8910E'
+        accentTertiary: '#E8910E',
+        backgroundColorSource: '#0093D0'
```

Comment:
i think you can just use accentPrimary anywhere you're currently using backgroundColorSource, so backgroundColorSource might be unnecessary. unless we plan on changing those colours separately sometime in the future